### ISSUE TITLE
SciPy-bundle-2020.11-{foss,fosscuda}-2020b build failing numpy test on CentOS 8.3 Intel CascadeLake

### ISSUE THREAD

**Issue by dithwick** [easybuilders/easybuild-easyconfigs#12648]
I've hit an odd issue that I've already found a solution for but thought I'd better report it.

I'm starting afresh with EasyBuild 4.3.4 (no prior software installed) on an Intel Cascade Lake cluster running CentOS 8.3. When building SciPy-bundle-2020.11-foss-2020b  or SciPy-bundle-2020.11-fosscuda-2020b on the login nodes (Cascade Lake Gold) it builds without complaint. When built on the compute nodes one of the numpy tests was failing:
```
core/tests/test_multiarray.py .........................................................................................xx....................................................................................................... [ 16%]
.......................s..............................................................................................................s.........................................................................Fatal Python error: Segmentation fault
```

This seemed to be consistent on both the standard compute and GPU nodes, both with Cascade Lake Platinum (in case it's relevant).

The fix to this (at least for fosscuda, haven't yet tested foss) was to change the OpenBLAS version for fosscuda and Scalapack from version 0.3.12 to 0.3.13. With this change SciPy-bundle-2020.11-fosscuda-2020b builds on the compute nodes without complaint. 

It occurred to me that the fix could be due to 

`	* Fixed a potentially severe memory leak after fork in OpenMP builds
	  that was introduced in 0.3.12`

as reported at https://www.openblas.net/Changelog.txt However it's still unclear to me as to why this consistently affected the compute/gpu nodes but not the login nodes.

**Comment by dithwick** [comment]
To confirm, swapping OpenBLAS version from 0.3.12 to 0.3.13 for SciPy-bundle-2020.11-foss-2020b (i.e. foss) also works.

Will check if https://github.com/easybuilders/easybuild-easyconfigs/pull/12649 fixes the issue.

**Comment by dithwick** [comment]
PR #12649  does appear to fix this issue.

**Comment by Micket** [comment]
Fixed by #12649
